core/src/store/function.rs

```diff
@@ -0,0 +1,32 @@
+use crate::data::CustomFunction;
+use crate::result::{Error, Result};
+use async_trait::async_trait;
+
+#[async_trait(?Send)]
+pub trait Function {
+    async fn fetch_function(&self, _func_name: &str) -> Result<Option<&CustomFunction>> {
+        Err(Error::StorageMsg(
+            "[Storage] Function is not supported".to_owned(),
+        ))
+    }
+    async fn show_functions(&self) -> Result<Vec<String>> {
+        Err(Error::StorageMsg(
+            "[Storage] Function is not supported".to_owned(),
+        ))
+    }
+}
+
+#[async_trait(?Send)]
+pub trait FunctionMut {
+    async fn create_function(&mut self, _func: CustomFunction) -> Result<()> {
+        Err(Error::StorageMsg(
+            "[Storage] Function is not supported".to_owned(),
+        ))
+    }
+
+    async fn drop_function(&mut self, _func_name: &str) -> Result<()> {
```

Comment:
As we use `insert` and `delete` prefixes in store traits, (e.g. StoreMut), how about `insert_function` and `delete_function`?